pkgs/tools/misc/bat-extras/modules/batpipe.nix

```
@@ -1,9 +1,21 @@
 {
+  lib,
+  stdenv,
   buildBatExtrasPkg,
   less,
+  procps,
 }:
 buildBatExtrasPkg {
   name = "batpipe";
-  dependencies = [ less ];
+  dependencies = [
+    less
+    procps
+  ];
+
+  patches = [
+    ../patches/batpipe-skip-outdated-test.patch
+  ]
+  ++ lib.optional stdenv.hostPlatform.isDarwin ../patches/batpipe-skip-detection-tests.patch;
```

```suggestion
  ++ lib.optionals stdenv.hostPlatform.isDarwin [ ../patches/batpipe-skip-detection-tests.patch ];
```
I’d suggest making consistent use of `lib.optionals`, this single `lib.optional` sticks out like a sore thumb 😅.